test.py

```diff
@@ -292,7 +292,7 @@ async def run(connection_string: str | None):
                                                                     break
                                                     
                                                     # Test disconnect tool if available
-                                                    break  # Exit schema loop once we've found a table
+                                                    # break  # Exit schema loop once we've found a table
```

Comment:
Can the `break` be removed?

Reply:
Please also note this PR is just for visibility purposes. It will probably never be merged, at least not in its current form.